src/lib.rs

```diff
@@ -626,13 +626,21 @@ pub enum Op {
     AssignUpvalue(usize),
 
     /// A helper instruction for the typechecker,
-    /// makes sure the top value on the stack
+    /// *makes sure* the top value on the stack
     /// is of the given type, and is ment to signal
     /// that the "variable" is added.
     /// (The type is looked up in the constants vector)
     ///
     /// Does not affect the stack.
     Define(usize),
+    /// A helper instruction for the typechecker,
+    /// *assumes* top value on the stack
+    /// is of the given type. Usefull for helping the
+    /// type system where it just can't do it.
+    /// (The type is looked up in the constants vector)
```

Comment:
```suggestion
    Define(usize),
    
    /// A helper instruction for the type checker.
    /// *Assumes* that the top value on the stack
    /// is of the given type. Useful for helping the
    /// type system where it just can't do it.
    /// (The type is looked up in the constants vector.)
```

Reply:
No, there's a grouping to this that is ruined by this formatting.